core/src/test/scala/akka/persistence/cassandra/PrintCreateStatements.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.persistence.cassandra
+
+import akka.persistence.cassandra.journal.CassandraJournalConfig
+import akka.persistence.cassandra.snapshot.CassandraSnapshotStoreConfig
+
+import akka.actor.ActorSystem
+
+/**
+ * Main application that prints the create keyspace and create table statements.
+ * It's using `cassandra-journal` and `cassandra-snapshot-store` configuration from default application.conf.
+ *
+ * These statements can be copy-pasted and run in `cqlsh`.
+ */
+object PrintCreateStatements {
+
+  def main(args: Array[String]): Unit = {
+    val system = ActorSystem("PrintCreateStatements")
```

Comment:
It would be nice if it didn't need an actor system but i don't think it is worth refactoring it not to